src/mmap.cpp

```diff
@@ -416,14 +414,11 @@ file_handle::file_handle(string_view name, std::int64_t const size
 	}
 #endif
 
-#ifdef POSIX_FADV_RANDOM
+#if (TORRENT_HAS_FADVISE && defined POSIX_FADV_RANDOM)
```

Comment:
interesting. android has the `POSIX_FADV_RANDOM` macro defined, but not the `posix_fadvice()` function?

Reply:
that's correct